packages/e2e-tests/tests/deploy-min.test.ts

```diff
@@ -0,0 +1,103 @@
+import { Blockchain } from "@rarible/api-client"
+import type { UnionAddress } from "@rarible/types"
+import { toUnionAddress } from "@rarible/types"
+import type { DeployTokenRequest } from "@rarible/sdk/src/types/nft/deploy/domain"
+import type { MintRequest } from "@rarible/sdk/build/types/nft/mint/mint-request.type"
+import type { BlockchainWallet } from "@rarible/sdk-wallet"
+import { getEthereumWallet, getTezosWallet, getWalletAddress } from "./common/wallet"
+import { createSdk } from "./common/create-sdk"
+import { mint } from "./common/atoms-tests/mint"
+import { getCollection } from "./common/helpers"
+import { deployCollection } from "./common/atoms-tests/deploy-collection"
+
+const suites: {
+	blockchain: Blockchain,
+	wallet: BlockchainWallet,
+	deployRequest: (address: UnionAddress) => DeployTokenRequest
+	mintRequest: (address: UnionAddress) => MintRequest
+}[] = [
+	{
+		blockchain: Blockchain.ETHEREUM,
+		wallet: getEthereumWallet(),
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		deployRequest: (walletAddress: UnionAddress) => {
```

Comment:
is it necessary to use deployRequest as a function?

Reply:
Yes, until it can depend on the arguments